### DISCUSSION TITLE
Fix Clang Error

### DISCUSSION THREAD

**Discussion by Prashant-1695** [ROM-builders/temporary#10051] [Q&A]
Someone used clang-14 and now it's conflicting with my kernel. Please Fix .
https://cirrus-ci.com/task/6110859901534208?logs=build#L3996

**Comment by Apon77** [comment]
> Someone used clang-14 and now it's conflicting with my kernel. Please Fix . https://cirrus-ci.com/task/6110859901534208?logs=build#L3996

how to fix?
